go.mod

```diff
@@ -25,18 +25,18 @@ require (
 	github.com/spf13/viper v1.7.0
 	github.com/stretchr/testify v1.7.0
 	github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5
-	go.opencensus.io v0.22.3
+	go.opencensus.io v0.23.0
 	golang.org/x/net v0.0.0-20211209124913-491a49abca63
 	golang.org/x/time v0.0.0-20210723032227-1f47c861a9ac
-	golang.org/x/tools v0.1.0
-	google.golang.org/genproto v0.0.0-20201110150050-8816d57aaa9a
-	google.golang.org/grpc v1.27.1
+	golang.org/x/tools v0.1.5
+	google.golang.org/genproto v0.0.0-20210402141018-6c239bbf2bb1
+	google.golang.org/grpc v1.36.1
```

Comment:
Just noting that if we are upgrading grpc, we will likely also want to regen all the clients at some point before the next release. Doesn't have to be in this PR, as long as it's noted in #2642

Reply:
Good point. I can tackle that in a follow up PR.